components/picker/index.css

```diff
@@ -54,7 +54,7 @@
 	--spectrum-picker-font-color-hover: var(--spectrum-neutral-content-color-hover);
 	--spectrum-picker-font-color-hover-open: var(--spectrum-neutral-content-color-focus-hover);
 	--spectrum-picker-font-color-active: var(--spectrum-neutral-content-color-down);
-	--spectrum-picker-font-color-key-focus: var(--spectrum-neutral-content-color-key-focus);
+	--spectrum-picker-font-color-key-focus: var(--spectrum-neutral-content-color-focus);
```

Comment:
This change was brought up by design (message in the thread: https://adobedesign.slack.com/archives/C06Q3G5HJUX/p1738865306488459?thread_ts=1738706953.819129&cid=C06Q3G5HJUX)

I'm working on clarifying `neutral-content-color-focus` vs. `netural-content-color-key-focus` (as key-focus was in the S2 picker token specs).